pkgs/by-name/go/google-chrome/package.nix

```diff
@@ -61,96 +63,103 @@
 
   # Additional dependencies according to other distros.
   ## Ubuntu
-  curl,
   liberation_ttf,
+  curl,
   util-linux,
-  wget,
   xdg-utils,
+  wget,
   ## Arch Linux.
   flac,
   harfbuzz,
   icu,
-  libopus,
   libpng,
+  libopus,
   snappy,
   speechd-minimal,
   ## Gentoo
   bzip2,
   libcap,
 
   # Necessary for USB audio devices.
-  libpulseaudio,
   pulseSupport ? true,
+  libpulseaudio,
 
-  adwaita-icon-theme,
   gsettings-desktop-schemas,
+  adwaita-icon-theme,
 
   # For video acceleration via VA-API (--enable-features=VaapiVideoDecoder)
-  libva,
   libvaSupport ? true,
+  libva,
 
   # For Vulkan support (--enable-features=Vulkan)
   addDriverRunpath,
+  common-updater-scripts,
+  jq,
+  libossp_uuid,
+  nix,
+  undmg,
 }:
 
 let
+  pname = "google-chrome";
+
   opusWithCustomModes = libopus.override { withCustomModes = true; };
 
   deps =
     [
-      alsa-lib
-      at-spi2-atk
-      at-spi2-core
-      atk
-      bzip2
-      cairo
-      coreutils
-      cups
-      curl
-      dbus
-      expat
-      flac
+      glib
```

Comment:
I don't know if you realize. But items were ordered alphabetically. I'd prefer if you kept them alphabetically ordered.